src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -1686,19 +1740,24 @@
 helps['network vnet'] = """
     type: group
     short-summary: Manage Azure Virtual Networks.
+    long-summary: To learn more about Virtual Networks visit https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-manage-network
 """
 
 helps['network vnet check-ip-address'] = """
     type: command
-    short-summary: Check if a private IP address is available for use.
+    short-summary: Check if a private IP address is available for use within a virtual network.
+    examples:
+        - name: Typical usage
```

Comment:
Bad description. Please keep example descriptions as short as possible, but precise. For example, here it would be `Check if the IP address 10.0.0.4 is available on the MyVnet network.` or similar (you may want to include wording to make it clearer that this is host-to-host networking, but that may be clear from context. Use your best judgement.)

Reply:
This comment must still be addressed.

Reply:
Fixed in my branch.